includes/definitions/ionos.yaml

```diff
@@ -0,0 +1,49 @@
+os: ionos
+text: 'Sagem ADR IONOS'
+type: network
+icon: sagemcom
+group: sagemcom
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'CPU Usage' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+discovery:
+    - sysObjectID:
+        - .1.3.6.1.4.1.1038
+poller_modules:
+            entity-physical: false
+            cisco-vrf-lite: false
+            storage: false
+            hr-mib: false
+            ucd-mib: false
+            arp-table: false
+            bgp-peers: false
+            vlans: false
```

Comment:
Please don't disable all these modules like this. Some of these don't even exist others won't actually cause any additional load.  You may add ones that have a tangible benefit.